mapie/multi_label_classification.py

```diff
@@ -59,7 +59,7 @@ class MapieMultiLabelClassifier(BaseEstimator, ClassifierMixin):
 
     random_state: Optional[Union[int, RandomState]]
         Pseudo random number generator state used for random uniform sampling
-        to evaluate quantiles and prediction sets in cumulated_score method.
+        to evaluate quantiles and prediction sets in ``"aps"`` method.
```

Comment:
Actually, this docstring is totally wrong as cumulated_score is never used in multilabel